dimagi/utils/tests/test_rate_limit.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime, timedelta
+from dimagi.utils.rate_limit import rate_limit
+from django.test import SimpleTestCase
+
+
+class RateLimitTestCase(SimpleTestCase):
+    def test_rate_limit(self):
+        start = datetime.utcnow()
+        rate_limit_count = 0
+        iteration_count = 0
+
+        while (datetime.utcnow() - start) < timedelta(seconds=18):
+            # Only allow 10 actions every 5 seconds in an 18 second period of time
+            if rate_limit('rate-limit-test', actions_allowed=10, how_often=5):
+                rate_limit_count += 1
+            iteration_count += 1
```

Comment:
I'm not sure if that would work - the expiration is set in redis and I don't think we'd be able to patch that.  If we wanted to reduce the test time, I could also just choose smaller numbers here.